### PR TITLE
5-serializers

### PR DESCRIPTION
**Pull request by JuankDawd**
# Description

In this Pull Request I upload the serializers for the `To Do` and `Item` models, now they look like this: 

image

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 class ItemsController < ApplicationController
+  before_action :authenticate_user!
   before_action :set_todo
   before_action :set_todo_item, only: %i[show update destroy]
-
+
```
Comment (luishurtado) [inline app/controllers/items_controller.rb:5]: 👀

File: app/controllers/todos_controller.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 class TodosController < ApplicationController
+  before_action :authenticate_user!
   before_action :set_todo, only: %i[show update destroy]
-
+
```
Comment (luishurtado) [inline app/controllers/todos_controller.rb:4]: 👀

**Review by luishurtado — CHANGES_REQUESTED** [review]

File: app/controllers/todos_controller.rb (hunk 2 above)
Comment (JuankDawd) [reply app/controllers/todos_controller.rb:4]: Solved running rubocop to get a nice and clean formmating

File: app/controllers/items_controller.rb (hunk 1 above)
Comment (JuankDawd) [reply app/controllers/items_controller.rb:5]: Solved running rubocop to get a nice and clean formmating

**Review by luishurtado — APPROVED** [review]

**Pull request merged**